src/unlimited.cpp

```diff
@@ -1814,8 +1829,22 @@ UniValue getstat(const UniValue &params, bool fHelp)
         }
         else
         {
+            //Get time before request
+            int64_t SeriesStartTime = GetTimeMillis();
             UniValue series = base->GetSeries(seriesStr, count);
+            int64_t SeriesEndTime = GetTimeMillis();
```

Comment:
Yeah this problem is why I originally thought we needed to get into the stat class. Also is why I captured a bracketed time (start and end) to indicate to the user if that has happened.